generate/resources/couchbase-edge-server/scripts/entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+if [ $# -ne 0 ]; then
+  if [ ! -f "$1" ]; then
+    echo "Error: Configuration file path is not valid."
+    exit 1
+  fi
+  exec couchbase-edge-server "$@"
```

Comment:
Is /opt/couchbase-edge-server/bin on the PATH?

Reply:
More generally: We need to determine the "API" for this. My proposal involved doing slightly different things based on whether the first argument was `cli`, `server`, or "something else", for instance. We don't have to do exactly that,  but we need to decide what the intended user experience is, and then write the entrypoint script to provide that.

Also, I think it's generally considered good practice to have an entrypoint script fall back to

    exec "$@"

at the end, if it doesn't recognize a specific command to be invoked - that allows people to do things like `docker run -it --rm couchbase/edge-server bash` etc. The `couchbase/server` entrypoint does that, for example.